clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1713,19 +1790,29 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
                 for (TopicPartition partition : partitions)
                     consumer.seek(partition, 0);
             }
+
+            @Override
+            public void onPartitionsLost(Collection<TopicPartition> partitions) {
+                System.out.println("hit partitions lost " + partitions);
+            }
         };
     }
 
     private ConsumerRebalanceListener getExceptionConsumerRebalanceListener() {
         return new ConsumerRebalanceListener() {
             @Override
             public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
-                throw new RuntimeException("boom!");
+                throw new RuntimeException("Hit partition revoke");
```

Comment:
Let's add the partitions parameter to the message as well and verify the expected partitions are revoked.